report-job-status/action.yml

```diff
@@ -40,14 +60,19 @@ runs:
           const baseUrl = "https://github.com"
           const eventName = "${{ github.event_name }}"
           const eventNumber = "${{ github.event.number }}"
+          const failedJobName = "${{ inputs.job-name }}" != '' ?
```

Comment:
Could you please check `"${{ inputs.job-name }}" || "${{ github.job }}"`?
It should work here.

Reply:
fixed